chombot-common/src/tournaments_watcher/notifier.rs

```diff
@@ -96,6 +95,11 @@ fn diff_as_message(diff: &TournamentStatus) -> String {
         TournamentStatus::Changed(change) => {
             str += &format!("**CHANGED**: _{}_; ", change.name);
 
+            if let Some(url) = &change.url {
+                if !url.is_empty() {
+                    str += &format!("website: {url}");
```

Comment:
I think you want to add `; ` in the string here, like with other updates? (or perhaps even ` ; ` so that you won't break the URL) Otherwise you might end up having the URL concatenated with some random stuff afterwards.

Reply:
omg true